pkg/utils/host/host.go

```diff
@@ -76,32 +74,12 @@ func Init(config Config) error {
 		return nil
 	}
 
-	// Apply workarounds
-	if autoSdUnitRestartFlag {
-		exit, err := autoSdUnitRestart()
-		if exit {
-			if err != nil {
-				fmt.Fprintf(os.Stderr, "error: %v\n", err)
-				os.Exit(1)
-			}
-			os.Exit(0)
-		}
-		if err != nil {
-			return err
-		}
-	}
 	if config.AutoMount {
 		err = autoMount()
 		if err != nil {
 			return err
 		}
 	}
-	if config.AutoWindowsWorkaround {
-		err = autoWindowsWorkaround()
-		if err != nil {
-			return err
```

Comment:
This workaround is for Docker Desktop on Windows; it's not related to the systemd workaround for kubectl-debug.